### DISCUSSION TITLE
Permissions with Security Groups

### DISCUSSION THREAD

**Discussion by andywingate** [StefanMaron/BusinessCentral.Sentinel#3] [Ideas]
With the move to the use of security groups, is it best practice now to avoid directly assigned permissions?

Warn if a user has any directly assigned permissions. Instead assign all permissions using security groups.

**Comment by StefanMaron** [comment]
Can you elaborate a bit on why that approach is better? 
Try to convince me like you would try to convince a customer 😁

That explanation will need to be part of the rule later anyway :)

**Comment by andywingate** [comment]
Happy to! 

When all permissions are assigned to a Security Group you can centrally manage permissions granted to users. With permissions assigned to users you need to maintain each user individually. 

It's the same reason its better to assign licenses and Azure roles to security groups and not directly to users. E.g. https://learn.microsoft.com/en-us/entra/identity/role-based-access-control/groups-concept

In the case of BC, if you avoid directly assigning permissions to users, it therefore takes the visibility of granted permissions to Entra/M365 admin, which is better for the IT folk!

Hope that helps!

Best Andy

**Reply by StefanMaron** [reply]
Thanks! I converted this to a discussion to track instead of an issue :)
